packages/gatsby/src/commands/clean.ts

```diff
@@ -10,7 +11,17 @@ import { IProgram } from "./types"
 module.exports = async function clean(program: IProgram): Promise<void> {
   const { directory, report } = program
 
-  const directories = [`.cache`, `public`]
+  const directories = [
+    `.cache`,
+    `public`,
+    // Ensure we clean babel loader cache
+    findCacheDir({
```

Comment:
what happens if this can't find anything - will it return undefined, false, null? And we would still call `fs.remove` on those? Not sure if that's potential problem or not, but maybe we should `.filter(Boolean)` on `directories` array to remove those?

Reply:
Yeah, that's a good idea. It returns undefined so filter(Boolean) makes sense